Makefile

```diff
@@ -47,6 +47,22 @@ bootstrap: ## Download and install all go dependencies (+ prep tooling in the ./
 	# prep temp dirs
 	mkdir -p tests/functional/output
 
+.PHONY: run-docker-registry
+run-docker-registry: bootstrap
+	# start a local registry
+	docker run -d -p 5000:5000 --name registry registry:2 | echo
+
+.PHONY: test
+test: run-docker-registry bootstrap
+	npm run build
+	./act -v -P ubuntu-latest=ghcr.io/catthehacker/ubuntu:js-latest -j test
```

Comment:
I may have missed it — are we explicitly installing `act` somewhere in this setup?

Reply:
yes, in `bootstrap`. the [python functional tests](https://github.com/anchore/scan-action/blob/main/.github/workflows/test.yml#L47) are [running `act` locally AND when run on github](https://github.com/anchore/scan-action/blob/main/Makefile#L27-L39). this is not ideal, but i didn't change this behavior. the new jest tests are being run [directly from npm on the github runner](https://github.com/anchore/scan-action/blob/main/.github/workflows/test.yml#L41)